README.md

```diff
@@ -1 +1,53 @@
-"# space-travelers-hub" 
+# Space Travelers Hub
+
+Web application that uses React and Redux to keep track of a list of missions ands rockets that the user can reserve and show them on their profile page.
+
+## Built With
+
+- Javascript
+- React
+- Redux
+- HTML
+- CSS
+
+### Prerequisites
+
+- Text Editor (VSCode for example)
+- Node and npm installed
+- Browser
+
+## Getting Started - How to deploy on your machine
```

Comment:
-[Optional] It would be nice that we try to add a live demo link for the project 😉